evalai/utils/challenges.py

```diff
@@ -0,0 +1,74 @@
+import os
+import json
+import requests
+
+from click import echo
+
+from pylsy import pylsytable
+
+from evalai.utils.auth import get_headers
+from evalai.utils.urls import urls
+from evalai.utils.common import valid_token
+
+
+API_HOST_URL = os.environ.get("EVALAI_API_URL", 'http://localhost:8000')
+
+
+def get_challenges(url):
+
+    headers = get_headers()
+    response = requests.get(url, headers=headers)
+    response_json = json.loads(response.text)
+
+    if valid_token(response_json):
+
+        column_names = ['ID', 'Challenge Name']
+        attributes = ['id', 'title']
+        table = pylsytable(column_names)
+
+        challenges = response_json["results"]
+        if len(challenges) is not 0:
+            for attribute, column_name in zip(attributes, column_names):
+                items = []
+                for challenge in challenges:
+                    items.append(challenge[attribute])
+
+                table.add_data(column_name, items)
+            echo(table)
+        else:
+            echo("Sorry, no challenges found.")
+    else:
+        echo("The authentication token you are using isn't valid. Please try again.")
+
+
+
+def get_challenge_list():
+    """
+    Fetches the list of challenges from the backend.
+    """
+    url = "{}{}".format(API_HOST_URL, urls["get_challenge_list"])
```

Comment:
I'm not a fan of direct dict calls like this, instead we should be using the `.get` method with a suitable default.
Another discussion point @deshraj @RishabhJain2018